challenges/binary-search/test_binary_search.py

```diff
@@ -3,36 +3,21 @@
 import binary_search
 
 
-def test_sorted_array_is_empty():
+def test_array_if_empty(sorted_array, test_value, low, high):
```

Comment:
The test should not have any parameters here as this will prevent the test from running.